cobrautil.go

```diff
@@ -173,28 +176,52 @@ func OpenTelemetryRunE(flagPrefix string, prerunLevel zerolog.Level) CobraRunFun
 
 		provider := strings.ToLower(MustGetString(cmd, prefixed("provider")))
 		serviceName := MustGetString(cmd, prefixed("service-name"))
+		endpoint, _ := cmd.Flags().GetString(prefixed("endpoint"))
```

Comment:
This should probably be `MustGetString(cmd, prefixed("endpoint"))`

Reply:
True, updated it so the function uses the same Method to get the Flag value.

Reply:
Sorry I missed this comment. This should not be `MustGetString`, since my intention was to make specifying the endpoint via the command line optional. The reasoning is `WithEndpoint()` does not allow configuration of the URL path, so if that is needed, configuration via environment variable is required.

Reply:
In my Test, the`MustGetString` will just render an empty String as that is set as default.